### ISSUE TITLE
Using the RHS to send a ToDo shows no feedback if the target user has disabled ToDo 

### ISSUE THREAD

**Issue by DHaussermann** [mattermost/mattermost-plugin-todo#184]
Using the RHS to send a ToDo shows no feedback if the target user has disabled ToDo. Sending via the slash command shows feedback that the user has blocked incoming ToDo but when a ToDo is created and assigned via the RHS the ToDo is simply not shown anywhere and the sender is not given any feedback about why. 

Steps:
- Login as a test user
- Type `/todo settings allow_incoming_task_requests off `
- Login as a 2nd user
- Create a new ToDO from the RHS assigned to the 1st user
**Observed**: ToDo save has no affect and ToDo is not visible anywhere
**Expected**: The UI should also show feedback that the target user has blocked incoming ToDo the way the slash command does

**Note**: This is not a new issue. It occurs on current production `v0.6.0`

**Comment by pradeepmurugesan** [comment]
Hi.. Would like to take a shot on this one.

**Comment by pradeepmurugesan** [comment]
@DHaussermann  couldn't reproduce the issue with the steps above. Getting a ping from the todo bot stating the user has blocked requests. Attaching screenshot.
![Screenshot 2022-10-10 at 07 50 52](https://user-images.githubusercontent.com/528317/194811631-44119578-42f6-49e9-9856-7e602bbd8fe8.png)

**Comment by DHaussermann** [comment]
@pradeepmurugesan You're correct. I'm not sure how I missed this before or what build I may have been testing on.  
On the latest build - I am  seeing a DM from the bot when I click `Save`.
Thank you for bringing this to my attention 🎉 

This is not ideal UX as it's not as obvious as feedback in the RHS but, the user does still get notified of the business logic.

I'm sorry for the invalid issue. Please ping if there's maybe another issue you would prefer to work on instead.
